_data/nav.yml

```diff
@@ -567,6 +568,7 @@
     - url: "configure"
     - url: "troubleshoot"
   - url: "advcode"
+  - url: "advanced_tables"
```

Comment:
This url is inconsistent with all other premium plugins (they use a hyphen instead of an underscore)

Reply:
updated